assets/css/main.css

```diff
@@ -106,6 +106,7 @@ body {
 }
 .work {
   background: #000;
+  background-size: 100%;
```

Comment:
Totally agree, working on sections now.  That was an initial idea I had when I started testing the page.  

j


> On May 4, 2016, at 5:25 AM, Alex Abbott <notifications@github.com> wrote:
> 
> In assets/css/main.css <https://github.com/alexabbott/blkbox-fullsite/pull/7#discussion_r62029487>:
> 
> > @@ -106,6 +106,7 @@ body {
> >  }
> >  .work {
> >    background: #000;
> > +  background-size: 100%;
> @jimijack <https://github.com/jimijack> I don't think we should use one huge background image for this. It should be broken into sections
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/alexabbott/blkbox-fullsite/pull/7/files/4a32c740b3d491131328a44b64a598bf70acf421..29adc1a38ef97d716ccc9900dfb351d8387a8538#r62029487>